### DISCUSSION TITLE
Can AST sign XML files?

### DISCUSSION THREAD

**Discussion by andyfisher100** [vcsjones/AzureSignTool#211] [Q&A]
Not too much to it really other than the question in the title, is it possible to use this tool to sign xml files? I had a look through the docs and tried it locally but not much joy

**Comment by KasperRagnar** [comment]
@andyfisher100 Currently, it looks like the AzureSignTool supports the following file formats:

.exe, .dll, .sys, .vxd, .scr, .efi, .msi, .msp, .msm, .cab, .appx, .appxbundle, .msix, .msixbundle, .ps1, .psm1
plus anything that uses standard Authenticode/PE signing. 

From what I understand, AzureSignTool still relies entirely on Windows Authenticode + SIPs. 
Windows only supports signing file types that have a registered SIP, and there is no SIP for XML in Windows.
